#! /bin/sh /usr/share/dpatch/dpatch-run
## 03quiet_start.dpatch by  <bernat@neo.luffy.cx>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Avoid displaying a lot of information when starting xrdp and sesman

@DPATCH@

--- xrdp-0.4.0.orig/sesman/config.c
+++ xrdp-0.4.0/sesman/config.c
@@ -167,13 +167,13 @@
   }
 
   /* showing read config */
-  g_printf("sesman config:\r\n");
+  /* g_printf("sesman config:\r\n");
   g_printf("\tListenAddress:            %s\r\n", cf->listen_address);
   g_printf("\tListenPort:               %s\r\n", cf->listen_port);
   g_printf("\tEnableUserWindowManager:  %i\r\n", cf->enable_user_wm);
   g_printf("\tUserWindowManager:        %s\r\n", cf->user_wm);
   g_printf("\tDefaultWindowManager:     %s\r\n", cf->default_wm);
-  g_printf("\tAuthFilePath:             %s\r\n", ((cf->auth_file_path) ? (cf->auth_file_path) : ("disabled")));
+  g_printf("\tAuthFilePath:             %s\r\n", ((cf->auth_file_path) ? (cf->auth_file_path) : ("disabled"))); */
 
   return 0;
 }
@@ -224,11 +224,11 @@
     lc->log_file=g_strdup("./sesman.log");
   }
 
-  g_printf("logging configuration:\r\n");
+  /* g_printf("logging configuration:\r\n");
   g_printf("\tLogFile:       %s\r\n",lc->log_file);
   g_printf("\tLogLevel:      %i\r\n", lc->log_level);
   g_printf("\tEnableSyslog:  %i\r\n", lc->enable_syslog);
-  g_printf("\tSyslogLevel:   %i\r\n", lc->syslog_level);
+  g_printf("\tSyslogLevel:   %i\r\n", lc->syslog_level); */
 
   return 0;
 }
@@ -283,7 +283,7 @@
   }
 
   /* printing security config */
-  g_printf("security configuration:\r\n");
+  /* g_printf("security configuration:\r\n");
   g_printf("\tAllowRootLogin:       %i\r\n",sc->allow_root);
   g_printf("\tMaxLoginRetry:        %i\r\n",sc->login_retry);
   if (sc->ts_users_enable)
@@ -301,7 +301,7 @@
   else
   {
     g_printf("\tNo TSAdminsGroup defined\r\n");
-  }
+    } */
 
   return 0;
 }
@@ -346,11 +346,11 @@
   }
 
   /* printing security config */
-  g_printf("session configuration:\r\n");
+  /* g_printf("session configuration:\r\n");
   g_printf("\tMaxSessions:                 %i\r\n", se->max_sessions);
   g_printf("\tKillDisconnected:            %i\r\n", se->kill_disconnected);
   g_printf("\tIdleTimeLimit:               %i\r\n", se->max_idle_time);
-  g_printf("\tDisconnectedTimeLimit:       %i\r\n", se->max_idle_time);
+  g_printf("\tDisconnectedTimeLimit:       %i\r\n", se->max_idle_time); */
 
   return 0;
 }
@@ -374,11 +374,11 @@
   }
 
   /* printing security config */
-  g_printf("X11rdp parameters:\r\n");
+  /* g_printf("X11rdp parameters:\r\n");
   for (i = 0; i < cs->rdp_params->count; i++)
   {
     g_printf("\tParameter %02d                   %s\r\n", i, (char*)list_get_item(cs->rdp_params, i));
-  }
+    } */
 
   return 0;
 }
@@ -402,11 +402,11 @@
   }
 
   /* printing security config */
-  g_printf("Xvnc parameters:\r\n");
+  /* g_printf("Xvnc parameters:\r\n");
   for (i = 0; i < cs->vnc_params->count; i++)
   {
     g_printf("\tParameter %02d                   %s\r\n", i, (char*)list_get_item(cs->vnc_params, i));
-  }
+    } */
 
   return 0;
 }
--- xrdp-0.4.0.orig/sesman/sesman.c
+++ xrdp-0.4.0/sesman/sesman.c
@@ -104,7 +104,7 @@
   if (1 == argc)
   {
     /* no options on command line. normal startup */
-    g_printf("starting sesman...\n");
+    /* g_printf("starting sesman...\n"); */
     daemon = 1;
   }
   else if ((2 == argc) && ((0 == g_strcasecmp(argv[1], "--nodaemon")) ||
--- xrdp-0.4.0.orig/xrdp/xrdp.c
+++ xrdp-0.4.0/xrdp/xrdp.c
@@ -480,7 +480,7 @@
     }
     if (0 != pid)
     {
-      g_writeln("process %d started ok", pid);
+      /* g_writeln("process %d started ok", pid); */
       /* exit, this is the main process */
       g_exit(0);
     }
